modules/EvseManager/Charger.cpp

```diff
@@ -15,6 +15,8 @@
 
 #include <fmt/core.h>
 
+using namespace std::chrono_literals;
```

Comment:
although it doesn't really hurt here, but extern namespaces should be made completely global, especially if there not used that much (only two times here)